libr/core/cmd_search.c

```diff
@@ -1924,6 +1924,9 @@ static void do_string_search(RCore *core, struct search_parameters *param) {
 		r_cons_printf ("fs hits\n");
 	}
 	core->search->inverse = param->inverse;
+	if (param->inverse) {
+		core->search->maxhits = 1;
```

Comment:
The old behavior is stopping at the first occurrence:

```c
-                       /* Stop at the first occurrence */
-                       if (s->inverse) {
-                               return -1;
-                       }
```

Then `r_search_update` returns `-1` and `do_string_search` in cmd_search.c breaks the loop.